schemas/v1.0/azure.yaml.json

```diff
@@ -27,6 +27,33 @@
                 }
             }
         },
+        "infra": {
+            "type": "object",
+            "title": "The infrastructure configuration used for the application",
+            "description": "Optional. Provides additional configuration for Azure infrastructure provisioning.",
+            "additionalProperties": true,
+            "properties": {
+                "provider":{
+                    "type": "string",
+                    "title": "Type of infrastructure provisioning provider",
+                    "description": "Optional. The infrastructure provisioning provider used to provision the Azure resources for the application. (Default: bicep)",
+                    "enum": [
+                        "bicep",
+                        "terraform"
+                    ]
+                },
+                "path": {
+                    "type": "string",
+                    "title": "Path to the location that contains Azure provisioning templates",
+                    "description": "Optional.The relative folder path to the Azure provisioning templates for the specified provider. (Default: infra)"
+                },
+                "module": {
```

Comment:
Just wondering - Does this have a semantics for terraform?  IIRC they just load all the `.tf` files from the folder, unlike bicep where there is a single entrypoint.

Reply:
Correct.